pkg/utils/syscalls/syscalls-helper.go

```diff
@@ -0,0 +1,46 @@
+package syscalls
+
+import (
+	"runtime"
+
+	amd64syscalls "github.com/inspektor-gadget/inspektor-gadget/pkg/utils/syscalls/amd64"
+	arm64syscalls "github.com/inspektor-gadget/inspektor-gadget/pkg/utils/syscalls/arm64"
+)
+
+func GetSyscallNumberByName(arch string, name string) (int, bool) {
+	if arch == "" {
+		arch = runtime.GOARCH
+	}
```

Comment:
I don't think we need this parameter. We're only interested in getting that information for the current architecture, so I'll remove this and use the conditional compilation logic described in my other comment.

Reply:
Done